src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Array.from(Object.values(robot));
```

Comment:
`Object.values({...})` і так повертає масив

```suggestion
  const values = Object.values(robot);
```

Reply:
Дякую, виправила!